### DISCUSSION TITLE
Terminal Chat – Feature Discussion!

### DISCUSSION THREAD

**Discussion by nguyen-dows** [microsoft/terminal#16454] [Ideas]
Please tell us what you think about Terminal Chat! We’d love to hear your feedback. 
Those who haven’t tried Terminal Chat are welcome to join this discussion as well! 

Please let us know: 
- What do you use Terminal Chat for? 
- Why did you choose to try Terminal Chat?
- What could be improved or added to this feature? 
- What would you not want to be added to this feature?
- What issues do you have using or trying to use this feature? 
- Why aren’t you using this feature?

Or anything else you’d like to talk about regarding Terminal Chat!

For now, Terminal Chat will be staying in Windows Terminal Canary, the nightly build of Windows Terminal that contains our most experimental features. We do not have any plans to push Terminal Chat to Windows Terminal Preview or Windows Terminal stable until we can provide it as an opt-in experience and identify the minimal loveable feature set that should go into our core product. 

We know “AI in a terminal” can sound daunting and we are always looking to improve 😊

We also have a scenario for Terminal Chat-related feature requests and bugs (#15000) so please take a look at it!
If there is anything that you would like us to add in our [documentation](https://learn.microsoft.com/en-us/windows/terminal/terminal-chat) to improve the setup experience, please let us know too!

**Comment by floh96** [comment]
> Why aren’t you using this feature?

I don't wnat to manually set it up

**Comment by 237dmitry** [comment]
> Why aren’t you using this feature?

Because:

> We do not have any plans to push Terminal Chat to Windows Terminal Preview

**Comment by calvinwonghk** [comment]
What do you use Terminal Chat for?  use to get the command easily, instead of reading doc or the result
Why did you choose to try Terminal Chat? just give a try
What could be improved or added to this feature? unable to use local model
What would you not want to be added to this feature? allow ollama, or lm studio
What issues do you have using or trying to use this feature? unable to use like result "The requested model is not supported."
Why aren’t you using this feature? unable to use it easily!
